lizmap/modules/lizmap/lib/Form/QgisForm.php

```diff
@@ -1762,6 +1762,7 @@ protected function PerformWfsRequest($wfsRequest, $formControl, $referencedField
             }
 
             // orderByValue
+            // orderByValue has been removed from XML since QGIS 3.32
             if ($formControl->relationReferenceData['orderByValue']) {
```

Comment:
```suggestion
            TODO Remove when QGIS 3.32 will be the minimum version for allowing a QGIS project
            if ($formControl->relationReferenceData['orderByValue']) {
```
I know, it doesn't exist yet. But something will arrive later about this I guess.

Reply:
Thanks @Gustry